Makefile

```diff
@@ -48,7 +48,7 @@ MANIFEST_DIR	?= /var/lib/nginx-agent
 DIRS            = $(BUILD_DIR) $(TEST_BUILD_DIR) $(BUILD_DIR)/$(DOCS_DIR) $(BUILD_DIR)/$(DOCS_DIR)/$(PROTO_DIR)
 $(shell mkdir -p $(DIRS))
 
-VERSION 		?= "v3.0.0"
+VERSION 		?= $(shell git describe --tags --abbrev=0)
```

Comment:
Can you follow the same logic as in V2?
https://github.com/nginx/agent/blob/dev-v2/Makefile#L6

There is an additional if statement needed for people that might fork the repo